docs/index.bs

```diff
@@ -2544,14 +2544,14 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
       : Input
       :: |job|, a [=job=]
-      :: |reason|, an [=exception=]
+      :: |errorName|, a string
       : Output
       :: none
 
-      1. If |job|'s [=job/client=] is not null, [=queue a task=], on |job|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |job|'s [=job/job promise=] with an exception that is a copy of |reason|, in |job|'s [=job/client=]'s [=environment settings object/Realm=].
+      1. If |job|'s [=job/client=] is not null, [=queue a task=], on |job|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |job|'s [=job/job promise=] with a [=exception/create|new=] [=exception=] with |errorName| and a user agent-defined [=exception/message=].
       1. For each |equivalentJob| in |job|'s [=list of equivalent jobs=]:
-          1. If |equivalentJob|'s [=job/client=] is null, continue to the next iteration of the loop:
-          1. [=Queue a task=], on |equivalentJob|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |equivalentJob|'s [=job/job promise=] with an exception that is a copy of |reason|, in |equivalentJob|'s [=job/client=]'s [=environment settings object/Realm=].
+          1. If |equivalentJob|'s [=job/client=] is null, [=iteration/continue=].
+          1. [=Queue a task=], on |equivalentJob|'s [=job/client=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to reject |equivalentJob|'s [=job/job promise=] with a [=exception/create|new=] [=exception=] with |errorName| and a user agent-defined [=exception/message=].
```

Comment:
Right. That structure didn't cover both cases. I tried it again on your comments. I left the "user agent-defined message" bit as-is. Would it make sense?